packages/utils/src/address.ts

```diff
@@ -12,12 +12,18 @@ export const privateKeyToAddress = (privateKey: string) => {
   )
 }
 
-export const publicKeyToAddress = (publicKey: string) => {
-  return '0x' + pubToAddress(Buffer.from(publicKey.slice(2), 'hex')).toString('hex')
+export const privateKeyToPublicKey = (privateKey: string) => {
+  return toChecksumAddress(
+    '0x' + privateToPublic(Buffer.from(privateKey.slice(2), 'hex')).toString('hex')
+  )
 }
 
-export const privateKeyToPublicKey = (privateKey: string) => {
-  return '0x' + privateToPublic(Buffer.from(privateKey.slice(2), 'hex')).toString('hex')
+export const publicKeyToAddress = (publicKey: string) => {
+  return toChecksumAddress(
+    '0x' + pubToAddress(Buffer.from(publicKey.slice(2), 'hex')).toString('hex')
+  )
 }
 
 export { toChecksumAddress } from 'ethereumjs-util'
+export { isValidAddress } from 'ethereumjs-util'
```

Comment:
Why are these re-exported here?
@nambrot FYI this conflicts with the changes you made in your Attestations svc PR

Reply:
They're used elsewhere, allows us to only depend on`@celo/utils` in those places rather than also `ethereumjs-utils`